docs/CONTRIBUTING-id.md

```diff
@@ -135,30 +135,30 @@ Contoh:
     BAIK  : * [Buku yang Sangat Bagus (1970)](https://example.org/book.html) - Jane Roe
     ```
 
-- <a id="in_process"></a>Buku dalam proses:
+- <a id="in_process"></a>Buku dalam proses penulisan:
 
     ```text
     BAIK  : * [Akan Segera Menjadi Buku yang Luar Biasa](http://example.com/book2.html) - John Doe (HTML) (:construction: *in process*)
     ```
 
-- <a id="archived"></a>Archived link:
+- <a id="archived"></a>Tautan archived:
```

Comment:
I think the translation below is better.

> 🇮🇩 Saya rasa terjemahan berikut lebih baik.

```suggestion
- <a id="archived"></a>Tautan yang diarsipkan:
```

Reply:
Resolved!